packages/calcite-components/src/components/action-bar/action-bar.stories.ts

```diff
@@ -148,6 +148,36 @@ export const darkModeRTL_TestOnly = (): string =>
 
 darkModeRTL_TestOnly.parameters = { modes: modesDarkDefault };
 
+export const withSlottedTooltips = (): string => html`<div style="display:flex; height:500px; width: 200px;">
```

Comment:
This story seems unrelated to the timing changes. Can you submit them in a separate `test` type PR?

Reply:
Its related to the timing changes to see instant tooltips once one is already open. Its the main request use case for instant tooltips.

Reply:
I see. Could you update the story name to reflect the main requested use case then?